### ISSUE TITLE
Time to add bismuth and antimony

### ISSUE THREAD

**Issue by angeltxilon** [Azanor/thaumcraft-suggestions#128]
I think it would be a good idea to **add bismuth and antimony**. These two metals would be complementing to the mercury, being used in similar applications to mercury or in alchemy (as catalyst, enhancer or raw material).

For example, antimony [{a heavy metal like lead that tends to form large crystals}] could have large amounts of ordo aspect, and thus be a good source or storage of ordo essence (this essence, according to my experience with the mod , is often difficult to obtain).
It could also be used for crafting "vis" ordo batteries, or batteries for other essences, something useful for recharging wands quickly in case of emergency.

Bismuth [{which usually crystallizes into spirals of cubes and prisms, and has iridescence}] could be used as a strong stabilizer in infusion altars, as well as a material for decorative blocks capable of changing color or to manufacture passive detectors of vis and/or appearance unknown.

**About its use in alchemy:**
Antimony could be a metal required for obtaining magical metals (thaumium and vacuum metal).
[{Inspired by the real-life purple alchemical alloy of antimony and copper}].

And bismuth could be added as an intermediate metal in the transmutation of mundane metals (iron, copper, lead, tin, brass) to gold and silver.
[{Bismuth was considered in times of alchemy as a half-formed metal between lead and silver}].

These both metals could be obtained by mining (ores of antimony and bismuth could have abundances similar or minor to gold (similar to cinnaber) or be generated in the nether instead overworld), or, in the opposite case, like byproduct of some alchemical conversions.

**Comment by carso1500** [comment]
yeah, bismuth has atributed many magical properties that it can have really interesting uses in the game

for example its said that it allow people to change forms and to contain "energies" inside of it, so it can be use as a weapon or a tool (maybe a multitool that changes forms) or like a special armour that allows you to disguise as mobs

**Comment by Vitzgraco** [comment]
It would be interesting to see more materials inside the mod.

**Comment by angeltxilon** [comment]
Polymorphic tools and armors of bismuth seems a good idea, carso1500.

**Comment by l-Luna** [comment]
Or maybe, like how brass is for alchemy, bismuth could be for Auromancy.
